### PR TITLE
Add CI for lint and unit tests

### LINKED ISSUE #3: add CI for lint and unit tests
add a CI using [github actions](https://github.com/Azure/eraser/actions/new) for basic checks

- [ ] linter - [golangci-lint action](https://github.com/golangci/golangci-lint-action)
- [ ] run unit tests

### PR DESCRIPTION
**Pull request by AldairCoronel**
Added the workflow.yaml

### CODE REVIEW

File: .github/workflows/workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: build_test
```
Comment (sozercan) [inline .github/workflows/workflow.yaml:1]:
```suggestion
name: test
```

we are not building yet, let's just change to test

File: .github/workflows/workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#     steps:
+#       - name: Set up Go 1.16
+#         uses: actions/setup-go@v2
+#         with:
+#           go-version: 1.16
+
+#       - name: Check out code into the Go module directory
+#         uses: actions/checkout@v2
+
+#       - name: Unit test
```
Comment (sozercan) [inline .github/workflows/workflow.yaml:35]: why is this commented?

Comment (AldairCoronel) [reply .github/workflows/workflow.yaml:35]: we don't have unit tests yet, right?

Comment (sozercan) [reply .github/workflows/workflow.yaml:35]: that's okay. it'll just pick it up once we have it. let's uncomment and add that

**Pull request closed without merging**